policyengine_us/variables/gov/states/mt/tax/income/mt_agi.py

```diff
@@ -13,7 +13,13 @@ def formula(person, period, parameters):
         agi = person("adjusted_gross_income_person", period)
         additions = person("mt_additions", period)
         subtractions = person("mt_subtractions", period)
-        mt_agi = max_(agi + additions - subtractions, 0)
+        reduced_agi = max_(agi + additions - subtractions, 0)
+        # Montana taxable social security benefits can be either addition or subtraction
+        # if mt_taxable_social security - taxable_social_security > 0, then addition, else subtraction
+        taxable_ss = person("taxable_social_security", period)
+        mt_taxable_ss = person("mt_taxable_social_security", period)
+        mt_ss_add_or_subtr = mt_taxable_ss - taxable_ss
+        mt_agi = max_(reduced_agi + mt_ss_add_or_subtr, 0)
```

Comment:
```suggestion
        tax_unit_mt_agi = max_(reduced_agi + mt_ss_add_or_subtr, 0)
```
To distinguish from the result given the name